base/.s2i/bin/assemble

```diff
@@ -37,6 +37,7 @@ jupyter labextension install @jupyter-widgets/jupyterlab-manager@1.1.0 \
                              jupyterlab-topbar-extension@0.4.0 \
                              jupyterlab-system-monitor@0.4.1 \
                              jupyterlab-plotly@4.14.3 \
+                             nglview-js-widgets@2.7.1 \
```

Comment:
@florist-gump  You can use the latest `nglview` with jupyterlab 3 to avoid rebuild the extension. 

https://github.com/nglviewer/nglview#installation